project-koku/koku
# [COST-2578] change report_periods to report_period_ids

Fixes [COST-2578](https://issues.redhat.com/browse/COST-2578)

Changes proposed in this PR:
*
*

Testing Instructions:
1. Run `this command` ...
2. Ingest this data `with this other command` ...
3. Expect to see this result:
   ```
   GET http://localhost:8000/api/endpoint

   {json: response}
   ```
4.
5.

---
Additional Context:
Talked with Eva about the 2 smoke failures here, she said its a known thing and unrelated: https://ci.ext.devshift.net/job/project-koku-koku-pr-check/4886/
